include/forkserver.h

```diff
@@ -167,6 +169,7 @@ typedef struct afl_forkserver {
   u8 *shmem_fuzz;                       /* allocated memory for fuzzing     */
 
   char *cmplog_binary;                  /* the name of the cmplog binary    */
+  char *asanfuzz_binary;                /* the name of the ASAN binary      */
```

Comment:
The original idea of forksever.h was to have one forkserver per binary, not to add all binaries to the same forkserver :D 
but I guess this already failed with cmplog

Reply:
We indeed have multiple forkservers here. See https://github.com/AFLplusplus/AFLplusplus/pull/2288/files/efa2120935a5105d1a0561bacf9f2fb3b7b92156#diff-58bda4a9d7e04051b895bec22791293b2c01b03b6adf79d56134f09e5d838333R739

I added this field because I don't want to reuse `cmplog_binary` for clarity. At this moment, I think SAND is not compatible with cmplog but it is perfectly doable. I would do it in a second PR.